.gitpod.yml

```diff
@@ -0,0 +1,34 @@
+tasks:
+  - name: Nats
+    init: |
+      docker-compose pull
+    command: |
+      docker-compose up -d
```

Comment:
Automatically starting this might be a problem:

```
WARN[0000] The "ADMIN_TOKEN" variable is not set. Defaulting to a blank string. 
WARN[0000] The "MANAGER_TOKEN" variable is not set. Defaulting to a blank string. 
WARN[0000] The "COLLECTOR_TOKEN" variable is not set. Defaulting to a blank string. 
WARN[0000] The "ADMIN_TOKEN" variable is not set. Defaulting to a blank string.
```

Reply:
Should we add some before step adding some env file?

Reply:
I added a script for copying the env file